reasoner_util/__init__.py

```diff
@@ -1,12 +1,34 @@
 """Build merging functions."""
 from typing import Hashable, Iterable, List, TypeVar
+import httpx
 
 T = TypeVar("T", bound=Hashable)
 
+
 def merge_iterables(cats1: Iterable[T], cats2: Iterable[T]) -> List[T]:
     """Merge iterables."""
     return list(set(cats1) | set(cats2))
 
 
 merge_ids = merge_iterables
 merge_categories = merge_iterables
+
+
+def get_preferred_ids(curies: List[str]) -> List[str]:
+    """Get list of preferred ids for a list of CURIES."""
+    params = {"curie": curies}
+    response = httpx.get(
+        "https://nodenormalization-sri-dev.renci.org/1.1/get_normalized_nodes",
+        params=params,
+    )
+    response_dict = response.json()
+    preferred_ids = [response_dict[curie]["id"]["identifier"] for curie in curies]
```

Comment:
If you want this shorter, this is a common convention:
```suggestion
    preferred_ids = [
        response_dict[curie]["id"]["identifier"]
        for curie in curies
    ]
```

Reply:
Perfect! thanks!